### PR TITLE
Add pr template, readme and .gitignore. Add the trivy workflow.

### PR DESCRIPTION
**Pull request by paivin-dn**
## Motivation
We've decided to use [the trivy tool](https://github.com/aquasecurity/trivy) as a vulnerabilities scanner. To avoid copy-paste workflows, we need to keep the workflow in one repository and include it in another repos through [reusable-workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows). 

## Solution
.gitignore, README.md and pull_request_template.md have been added as standard files for our projects. 
`Trivy` pipeline has been added and tested in another repository.

## Checklist

<details open=true>
  <summary>PR checklist (self-check before review)</summary>

- [X] The branch name fits the `pr-<prefix>-<task-name>` pattern;
- [X] The PR has a meaningful name using the `<prefix>: <description>` pattern;
- [X] The PR is properly labeled depending on the change;
- [X] The PR is assigned to you;
- [X] If the PR is based on another PR, links to them are added to the PR header and the "has PR dependency" label is set;
- [X] Self-review. Review your code, commits, and the PR description;
- [X] Make sure that you updated the corresponding docs in the `docs/` directory;
- [X] Links to the updated docs are added to the PR header (if any);
- [X] CI successfully finished;

</details>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--- Optional: PR dependencies
+
+> ⚠️ Depends on the following PRs, which have to be merged first: #2 #3
+
+--->
+
```
Comment (iamsaywhat) [inline .github/pull_request_template.md:6]:
```suggestion

📝 **ClickUp:** *link1*, *link2*, ...

<!--- Optional: other useful headers if applied

📓 **Docs**: [filename.md](link-to-docs/)

--->

```

**Review by iamsaywhat — APPROVED** [review]
Looks good, I hope it works 😄

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Motivation
+Explain a problem
+
+## Solution
+Describe a solution
+
+<!--- Optional: When changing dashboards, you need to attach screenshots of the changes.
```
Comment (anastasiia-garnaga) [inline .github/pull_request_template.md:13]: Nit: dashboards? Are we going to have them in these repo's PRs?
And do we really need screenshots in the PR description?

**Review by anastasiia-garnaga — APPROVED** [review]
GJ!

File: .github/pull_request_template.md (hunk 2 above)
Comment (paivin-dn) [reply .github/pull_request_template.md]: Removed it

**Review by glebignatieff — APPROVED** [review]
GJ! It's awesome that we don't have to copy-paste this workflow to all our repos :)

**Pull request merged**